templates/error_page.hbs

```diff
@@ -1,6 +1,5 @@
 <div class="container-divider"></div>
-<div class="error-page">
-  <h1>{{t 'oops'}}</h1>
+<div class="error-page text-center">
```

Comment:
I think the error page needs more spacing at the top of the page

maybe we can do 
.error-page {margin-top: 60px;}